sale_order_packaging_info/models/sale_order.py

```diff
@@ -0,0 +1,44 @@
+# SPDX-FileCopyrightText: 2023 Coop IT Easy SC
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    sale_order_packaging_line_ids = fields.One2many(
+        comodel_name="sale.order.packaging.line",
+        inverse_name="sale_order_id",
+        string="Packaging Lines",
+    )
+    packaging_amount_total = fields.Monetary(
+        string="Packaging Total", compute="_compute_packaging_amount"
+    )
+    packaging_other_text = fields.Text(string="Other (Packaging)")
```

Comment:
i’m not sure about the name of this field. i would rather use `additional` instead of `other`. or maybe something like `packaging_extra_info`. it feels strange to me to have a field called “other text”.

Reply:
Renamed to `packaging_notes`